README.md

```diff
@@ -32,6 +32,16 @@ Our chat channel is on gitter here: https://gitter.im/fossasia/susi_android
     <td><img src="docs/_static/message.png" height = "480" width="270"></td>
   </tr>
   <tr>
+    <td><img src="docs/_static/voice_input.png" height = "480" width="270"></td>
+    <td><img src="docs/_static/skills.png" height = "480" width="270"></td>
+    <td><img src="docs/_static/skills2.png" height = "480" width="270"></td>
```

Comment:
Too many. Not needed. Replace the third one in this row with second one on the next row and remove rest. i.e replace line 37 with line 41 and remove lines 39 to 43.

Reply:
On it.